tests/test_wave_01.py

```diff
@@ -51,27 +51,24 @@ def test_get_task(client, one_task):
     }
 
 
-@pytest.mark.skip(reason="No way to test this feature yet")
+
 def test_get_task_not_found(client):
     # Act
     response = client.get("/tasks/1")
     response_body = response.get_json()
 
     # Assert
     assert response.status_code == 404
+    assert response_body == {'message': 'Task.title 1 not found'}
```

Comment:
Here's where that comment I left about `cls.__name__` comes full circle. You want the model name `Task` or `Goal` as opposed to `Task.title` or `Goal.title`.